lib/Widgets/SettingsPage/AbstractSettingsPage.vala

```diff
@@ -21,15 +21,59 @@
  * AbstractSettingsPage is a #Gtk.ScrolledWindow subclass with properties used
  * by other Granite settings widgets.
  */
-
 public abstract class Granite.SettingsPage : Gtk.ScrolledWindow {
```

Comment:
I know this branch didn't change it, but this class should be within `Widgets` namespace.